src/main/java/org/example/homework9/Bed.java

```diff
@@ -0,0 +1,41 @@
+package org.example.homework9;
+
+public class Bed {
+    private String style;
+    private int pillows;
+    private int height;
+    private int sheets;
+    private int quilt;
+
+    public Bed(String style, int pillows, int height, int sheets, int quilt) {
+        this.style = style;
+        this.pillows = pillows;
+        this.height = height;
+        this.sheets = sheets;
+        this.quilt = quilt;
+    }
+
+    public void make() {
+        System.out.println("The bed is being made.");
```

Comment:
```suggestion
        System.out.println("Bed: The bed is being made.");
```
No sense to log identical messages in Bed and Bedroom class. Better to change in order to be able to track